src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+
+import Button from 'app/components/button';
+import CommitRow from 'app/components/commitRow';
+import {Panel} from 'app/components/panels';
+import {promptsUpdate} from 'app/actionCreators/prompts';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+
+const SAMPLE_COMMIT = {
+  id: 'ff821e0',
+  author: {name: 'User Name'},
+  dateCreated: moment()
+    .subtract(3, 'day')
+    .format(),
+  repository: {
+    provider: {id: 'integrations:github', name: 'GitHub', status: 'active'},
+  },
+  message: 'ref(oops): This commit accidentally broke something',
+};
+
+class EventCauseEmpty extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    project: SentryTypes.Project.isRequired,
+  };
+
+  state = {
+    shouldShow: undefined,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    const {api, project, organization} = this.props;
+
+    api.request('/promptsactivity/', {
+      method: 'GET',
+      query: {
+        project_id: project.id,
+        organization_id: organization.id,
+        feature: 'suspect_commits',
+      },
+      success: (data, _, jqXHR) => {
+        this.setState({
+          shouldShow: this.shouldShow(data),
+        });
+      },
+      error: err => {
+        Sentry.captureException(err);
+      },
+    });
+  }
+
+  shouldShow({data} = {}) {
+    if (data && data.dismissed_ts) {
+      return false;
+    }
+    if (data && data.snoozed_ts) {
+      // snoozed_ts is stored in seconds
+      const nowSeconds = Date.now() / 1000;
+      const snoozedDays = (nowSeconds - data.snoozed_ts) / (60 * 60 * 24);
```

Comment:
You could use [moment.subtract](https://momentjs.com/docs/#/manipulating/subtract/) or [diff & duration](https://momentjs.com/docs/#/durations/diffing/) to make this a bit easier to read and harder to make typos/mistakes on.